apertium-uzb-kaa.uzb-kaa.dix

```diff
@@ -463,12 +463,12 @@
 <e>       <p><l>bayram<s n="n"/></l>                   <r>bayram<s n="n"/></r></p></e>
 <e>       <p><l>baza<s n="n"/></l>                     <r>baza<s n="n"/></r></p></e>
 <e>       <p><l>belgi<s n="n"/></l>                    <r>belgi<s n="n"/></r></p></e>
-<e>       <p><l>berkin<s n="v"/><s n="iv"/></l>        <r>jasırın<s n="v"/><s n="iv"/></r></p></e>
+<!-- <e>       <p><l>berkin<s n="v"/><s n="iv"/></l>        <r>jasırın<s n="v"/><s n="iv"/></r></p></e> -->
 <e>       <p><l>beton<s n="n"/></l>                    <r>beton<s n="n"/></r></p></e>
 <e>       <p><l>bez<s n="n"/></l>                      <r>bez<s n="n"/></r></p></e>
-<e>       <p><l>bilim<s n="n"/></l>                    <r>bilim<s n="n"/></r></p></e>
-<e>       <p><l>bil<s n="v"/><s n="tv"/></l>           <r>bil<s n="v"/><s n="tv"/></r></p></e>
-<e>       <p><l>birlik<s n="n"/></l>                   <r>birlik<s n="n"/></r></p></e>
+<!-- <e>       <p><l>bilim<s n="n"/></l>                    <r>bilim<s n="n"/></r></p></e> -->
+<!-- <e>       <p><l>bil<s n="v"/><s n="tv"/></l>           <r>bil<s n="v"/><s n="tv"/></r></p></e>
+<e>       <p><l>birlik<s n="n"/></l>                   <r>birlik<s n="n"/></r></p></e> -->
```

Comment:
I don't understand what's wrong with these three words.  Are they not correct translations?

Reply:
they exists already, I just commented it out to verify if I can delete it.

Reply:
Okay.  If they're true duplicates then just delete them.